static/app/views/onboarding/documentationSetup.tsx

```diff
@@ -205,7 +205,7 @@ const Content = styled(motion.div)`
 
   .alert h5 {
     font-size: 1em;
-    margin-bottom: 1rem;
+    margin-bottom: 0.625rem;
```

Comment:
Scaling down by a factor of `10/16 = 0.625`.

With new value:
<img width="948" alt="Screen Shot 2021-12-16 at 11 23 27 AM" src="https://user-images.githubusercontent.com/44172267/146445440-1f0a6402-aa21-497d-88de-7c4ee77d2e84.png">